src/test/java/uk/nhs/hee/tis/revalidation/connection/service/GmcClientServiceTest.java

```diff
@@ -126,6 +127,31 @@ void shouldSubmitTryAddDoctorRequest() {
     assertThat(gmcConnectionResponseDto.getReturnCode(), is(returnCode));
   }
 
+  @Test
```

Comment:
If you think we reasonably might get a `null` here, please parameterise/add a test.

Reply:
So far we haven't from the GMC, even in the case of them returning an exception code they still pass an empty string